docs(changelog): 5.3 release (#1053)
stoplightio/spectral: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [5.3.0] - 2019-04-01

### Added

> - Built-in functions can now be accessed in custom functions [#925](https://github.com/stoplightio/spectral/pull/925)

Reviewer: ```suggestion
- Built-in functions can now be accessed from custom functions [#925](https://github.com/stoplightio/spectral/pull/925)
```

- CLI: `--show-unmatched-globs` flag  [#747](https://github.com/stoplightio/spectral/issues/747)

### Changed

> - `typed-enum` function is a part of the OpenAPI ruleset [#924](https://github.com/stoplightio/spectral/issues/924)

Reviewer: ```suggestion
- `typed-enum` function is no longer a built-in function. It's been migrated into a custom function of the OpenAPI ruleset [#924](https://github.com/stoplightio/spectral/issues/924)
```

> - `openapi-tags` rule fit its description [#1038](https://github.com/stoplightio/spectral/pull/1038)

Reviewer: ```suggestion
- `openapi-tags` rule has been fixed to make it fit its description [#1038](https://github.com/stoplightio/spectral/pull/1038)
```
